src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1777,6 +1781,54 @@ class Class
     //]
     }}
 }}
+";
+
+        private static readonly string s_allow_bank_line_after_token_in_conditional_expression_true = $@"
+class Class
+{{
+//[
+    // {ServicesVSResources.Allow_colon}
+    void Method()
+    {{
+        var result = CheckCondition() ?
+            WhenTrue :
+            WhenFalse;
+    }}
+//]
+}}
+";
+
+        private static readonly string s_allow_bank_line_after_token_in_conditional_expression_false = $@"
+namespace NS1
+{{
+    class Class
+    {{
+    //[
+        // {ServicesVSResources.Require_colon}
```

Comment:
💭 Require seems backward here. The example shows omitted.

Reply:
the pattern we have shows the form we require, versus the alternate.  So this is saying "you will be required to write it this way"